modes/longitudinal/src/index.js

```diff
@@ -1,38 +1,54 @@
-import toolbarButtons from './toolbarButtons.js';
 import { hotkeys } from '@ohif/core';
+import toolbarButtons from './toolbarButtons.js';
+import { id, version } from './id.js';
 
 const ohif = {
-  layout: 'org.ohif.default.layoutTemplateModule.viewerLayout',
-  sopClassHandler: 'org.ohif.default.sopClassHandlerModule.stack',
-  hangingProtocols: 'org.ohif.default.hangingProtocolModule.default',
+  layout: '@ohif/extension-default.layoutTemplateModule.viewerLayout',
+  sopClassHandler: '@ohif/extension-default.sopClassHandlerModule.stack',
+  hangingProtocols: '@ohif/extension-default.hangingProtocolModule.default',
 };
 
 const tracked = {
-  measurements: 'org.ohif.measurement-tracking.panelModule.trackedMeasurements',
-  thumbnailList: 'org.ohif.measurement-tracking.panelModule.seriesList',
-  viewport: 'org.ohif.measurement-tracking.viewportModule.cornerstone-tracked',
+  measurements:
+    '@ohif/extension-measurement-tracking.panelModule.trackedMeasurements',
+  thumbnailList: '@ohif/extension-measurement-tracking.panelModule.seriesList',
+  viewport:
+    '@ohif/extension-measurement-tracking.viewportModule.cornerstone-tracked',
 };
 
 const dicomsr = {
-  sopClassHandler: 'org.ohif.dicom-sr.sopClassHandlerModule.dicom-sr',
-  viewport: 'org.ohif.dicom-sr.viewportModule.dicom-sr',
+  sopClassHandler: '@ohif/extension-dicom-sr.sopClassHandlerModule.dicom-sr',
+  viewport: '@ohif/extension-dicom-sr.viewportModule.dicom-sr',
 };
 
 const dicomvideo = {
-  sopClassHandler: 'org.ohif.dicom-video.sopClassHandlerModule.dicom-video',
-  viewport: 'org.ohif.dicom-video.viewportModule.dicom-video',
-}
+  sopClassHandler:
+    '@ohif/extension-dicom-video.sopClassHandlerModule.dicom-video',
+  viewport: '@ohif/extension-dicom-video.viewportModule.dicom-video',
+};
```

Comment:
I know this wasn't changed in this PR, but its just come to my attention. Why are we adding dicom video and pdf to the longitudinal mode when these weren't in the original spec?

We can make a kitchen sink mode to demo everything, but the longitudinal mode has a very specific purpose/is a great of example of an actual mode rather than a swiss army knife. I'm not sure we should bloat it with new features? It shouldn't be the default mode to throw all shinies in?

cc @swederik

Reply:
that makes sense to me, i'm removing it